src/azure-cli/azure/cli/command_modules/security/tests/latest/test_iot_scenario.py

```diff
@@ -0,0 +1,27 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import ScenarioTest
```

Comment:
please make sure all new commands included in test.

Reply:
the other commands are alerts and recommendations
alerts get deleted after 14 days 
recreating alerts on a regular basis on an unknown subscription is problematic  
I consulted with ASC and they did the same on their alerts

Reply:
actually we have a test subscription. If there is special policy or service to config, we can add it in our test subscription.

Reply:
in order to test the alerts and recommendation, we need to simulate an attack on a specific device on always live IoT hub with our solution on continuously
it is not an easy task and even if we do that the test can take up to an hour or creating a side service that we need to maintain. (ASC didn’t do that for that reason)

Reply:
I see.